views/install.sh.erb

```diff
@@ -26,6 +26,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+download_timeout=10
```

Comment:
@lamont-granquist @cheeseplus You're right, it is quite low.  It may be worth the effort to allow 0 which doesn't set any timeouts, and use that by default.  That or pull the timeout functionality entirely (as lamont said, if you're truly offline these commands will fail without a timeout anyway).